CHANGES.txt

```diff
@@ -4,6 +4,7 @@ New in v8.1.1
 Bugfixes
 --------
 
+* Allow else clause in post-list plugin. (Issue #3436)
```

Comment:
This should be in the “new in master” block, please rebase on origin/master and fix the changelog.

Reply:
Sorry, what's the “new in master” block? I can't find a section like that.

Reply:
You’re looking at an outdated copy of this file, pull from upstream.

![image](https://user-images.githubusercontent.com/327323/87361533-bad6ca80-c56c-11ea-9dfb-4c5ded7b75fb.png)

Reply:
Ah! Got it. Thanks.